### ISSUE TITLE
[f7-checkbox] simple f7-checkbox has styling issues within "Form-Inputs"

### ISSUE THREAD

**Issue by nimo23** [framework7io/framework7#2148]
This is a (multiple allowed):
* [x] enhancement
* [ ] feature-discussion (RFC)

* Framework7 Version: 2 for **md and ios theme**

F7-checkbox is a good alternative for F7-toggle. However, when using F7-checkbox within Form-Inputs with this layout:

test1:
![checkbox](https://user-images.githubusercontent.com/3045549/35401924-42835828-01fb-11e8-9d2f-3a0c6e6e57a1.png)

or with this (test2):
![checkbox2](https://user-images.githubusercontent.com/3045549/35402687-a205e494-01fd-11e8-8e3b-23b0dcd95ffb.png)

**Small visual issues are there:**
- in "test1", ripple effect is not bounded
- in "test2", ripple effect is not within the whole list-element (setting width=100 does not help)
- styling (font/color) of label-text within f7-checkbox is not the same as other form-inputs
- if "info" is used within f7-checkbox, then "info"-text is not aligned to f7-checkbox
- when using "inline-labels" and putting "label-text" on the left side of the checkbox then it is not aligned with the other input-labels within the same `<ul>..</ul>`. 
- I can put the label outside of checkbox (_label left+checkbox right_) and use `<label=for`, so clicking the row activates the checkbox, however, there are styling issues.

Looking at http://framework7.io/docs/inputs.html, f7-checkbox seems not supported within Form-Inputs. However, besides the above styling issues all works. So would be nice to support the input-type "f7-checkbox" within "form-inputs" by solving the small visual issues above. 

**In short: support f7-checkbox out of the box within f7-form-inputs.**

Actually, I have to use the F7-toggle, but a **checkbox** is much simpler to use..
